src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -157,6 +157,9 @@ export class HuiDialogEditCard extends LitElement {
       value: cardConfig,
     };
     this._originalConfigYaml = cardConfig;
+
+    // This will center the dialog with the updated config Element
+    fireEvent(this._dialog, "iron-resize");
```

Comment:
We should only do this after the card has updated, which is what we used to do in `showDialog`

Reply:
I put this in the updated before and you said we shouldnt do that 🤔

Reply:
hmm. I don't remember the context, but I might have been referring to waiting till the config has been loaded from the backed, but I don't really know anymore.

Reply:
You said it would jump around too much in the `updated` method. But you may be referring to a different place to put it

Reply:
Ah yes, we shouldn't fire it in updated **every** time. But we can fire it in updated conditionally, for example if the config has been loaded and rendered.

Reply:
This may be better to let Bram do as he is reworking that file